python/llm/example/GPU/HF-Transformers-AutoModels/Advanced-Quantizations/GGUF-IQ2/README.md

```diff
@@ -0,0 +1,81 @@
+# GGUF-IQ2
+
+This example shows how to directly run GGUF-IQ2 models using BigDL-LLM on Intel GPU.
```

Comment:
how to run INT2 models using the IQ2 mechanism (first implemented by llama.cpp) in BigDL-LLM

Reply:
Have updated to this : )